GTG/gtg.in

```diff
@@ -99,8 +99,17 @@ if __name__ == "__main__":
 
         # Set up UI i18n
         LOCALE_DIR = '@localedir@'
-        locale.bindtextdomain('gtg', LOCALE_DIR)
-        locale.textdomain('gtg')
+
+        try:
+            locale.bindtextdomain('gtg', LOCALE_DIR)
+            locale.textdomain('gtg')
+        except AttributeError as e:
+            # Python built without gettext support doesn't have bindtextdomain()
+            # and textdomain(), so gracefully fail here.
+            print(
+                "Couldn't bind the translation domain. Some translations won't "
+            "work.\n{}".format(e))
```

Comment:
don't excessively newline here - we don't have any 80 character limit (at least not anymore). Up to 120 should be fine, this would be below 100. This feels like a rather technical message already, so I would specify "**gettext** translation domain"

Reply:
Thanks for the review :)